src/fastoad/notebooks/02_explanation/03_scipy.ipynb

```diff
@@ -0,0 +1,388 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    # could thus accelrate the process, we will start at a different value to test the function.
</pre><p>accel<strong>e</strong>rate</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Corrected in a03fe54a